src/EUPL-1.0.xml

```diff
@@ -0,0 +1,249 @@
+             Licence and the way it may be accessible, concluded, stored and reproduced by the
+             Licensee.</p>
+        </li>
+        <li>
+          <b>12.</b>
+          <p>Termination of the Licence</p>
+          <p>The Licence and the rights granted hereunder will terminate automatically upon any breach by the
+             Licensee of the terms of the Licence.</p>
+          <p>Such a termination will not terminate the licences of any person who has received the Work from
+             the Licensee under the Licence, provided such persons remain in full compliance with the
+             Licence.</p>
+        </li>
+        <li>
+          <b>13.</b>
+          <p>Miscellaneous</p>
+          <p>Without prejudice of Article 9 above, the Licence represents the complete agreement between the
+             Parties as to the Work licensed hereunder.</p>
+          <p>If any provision of the Licence is invalid or unenforceable under applicable law, this will not
+             affect the validity or enforceability of the Licence as a whole. Such provision will be
+             construed and/or reformed so as necessary to make it valid and enforceable.</p>
+          <p>The European Commission may put into force translations and/or binding new versions of this
+             Licence, so far this is required and reasonable. New versions of the Licence will be published
+             with a unique version number. The new version of the Licence becomes binding for You as soon
+             as You become aware of its publication.</p>
+        </li>
+        <li>
+          <b>14.</b>
+          <p>Jurisdiction</p>
+          <p>Any litigation resulting from the interpretation of this License, arising between the European
+             Commission, as a Licensor, and any Licensee, will be subject to the jurisdiction of the Court
+             of Justice of the European Communities, as laid down in article 238 of the Treaty establishing
+             the European Community.</p>
+          <p>Any litigation arising between Parties, other than the European Commission, and resulting from
+             the interpretation of this License, will be subject to the exclusive jurisdiction of the
+             competent court where the Licensor resides or conducts its primary business.</p>
+        </li>
+        <li>
+          <b>15.</b>
+          <p>Applicable Law</p>
+        </li>
+      </list>
+      <p>This Licence shall be governed by the law of the European Union country where the Licensor resides or has
+         his registered office.</p>
+      <p>This licence shall be governed by the Belgian law if:</p>
+      <p>− a litigation arises between the European Commission, as a Licensor, and any Licensee;</p>
+      <p>− the Licensor, other than the European Commission, has no residence or registered office inside a
+         European Union country.</p>
+    </body>
+    <optional>
```

Comment:
I think I would tend to make this optional, as @myndzi had suggested, but since it's not optional [here](http://git.spdx.org/?p=license-list.git;a=blob;f=EUPL-1.0.txt;h=5b1a6e6ad572c611cba0183aa2ccc4ee749837d4;hb=HEAD), I say we leave it as not optional for now and move on.